site/content/en/docs/Integration Patterns/player-capacity.md

```diff
@@ -8,8 +8,9 @@ description: >
 ---
 
 {{< alpha 
-    title="Player Tracking, Allocation Player Filter, and Allocation State Filter" 
-    gate="PlayerTracking,PlayerAllocationFilter,StateAllocationFilter" >}}
+    title="Player Tracking and Allocation Player Filter" 
```

Comment:
These changes will also need to be `feature` shortcoded, such that they only get updated when we do our next release.

Reply:
I added `feature` shortcode.